src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,15 +1,123 @@
-import { Component } from 'react';
+import React, { Component } from 'react';
+import './NewMovie.scss';
 
-type Props = {};
-type State = {};
+type Props = {
+  addMovie: (movie: Movie) => void,
+};
+
+type State = {
+  title?: string,
+  description?: string,
+  imgUrl?: string,
+  imdbUrl?: string,
+  imdbId?: string,
```

Comment:
do not cheat with TS)

```suggestion
  title: string,
  description: string,
  imgUrl: string,
  imdbUrl: string,
  imdbId: string,
```